service/src/main/resources/api/service_openapi.yaml

```diff
@@ -2417,6 +2531,17 @@ components:
           description: The ID of the Data Repo snapshot
           type: string
 
+    UpdateGitRepoReferenceRequestBody:
+      type: object
+      properties:
+        name:
+          $ref: "#/components/schemas/Name"
+        description:
+          type: string
+        gitCloneUrl:
```

Comment:
What makes this a "clone" URL and not just the git URL? Won't users do other things with it besides clone?

Reply:
this was changed from gitUrl based on Jay's previous comment because we can later support passing in git url for a specific branch or file. 

What can user do besides cloning though? I'm lack of a better word.. gitUrlMain to imply that it's from the main branch? gitUrlDefault?

Reply:
Maybe gitRepoUrl? Because it is pointing at the whole repo?

Reply:
I like gitRepoUrl.

Reply:
renamed.